.github/workflows/release.yml

```diff
@@ -0,0 +1,70 @@
+name: Build Releases
+
+on:
+  release:
+    types: [published]
+  push:
+    branches:
+      - 'master'
+      - 'develop'
+
+jobs:
+  release:
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+            os: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+      - name: "Create filename"
+        id: fn
+        shell: bash
+        run: echo "FILENAME=build/rctab_${{ github.ref_name }}_${{ runner.os }}.zip" >> $GITHUB_OUTPUT
+          
+      - uses: actions/checkout@v3
+
+      - name: "Set up JDK 17.0.2"
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17.0.2'
+          distribution: 'temurin'
+
+      - name: "Validate Gradle wrapper"
+        uses: gradle/wrapper-validation-action@ccb4328a959376b642e027874838f60f8e596de3
```

Comment:
I assume it'll stop the auto-build if this fails? Will we get emailed a notification or how exactly does that work?

Reply:
1. The release creator will get an email notification if it failed while creating a release
2. It will mark the branch as having a failed build + we'll get an email notification if it happens after a merge to master/develop, like so:
 
<img width="906" alt="image" src="https://user-images.githubusercontent.com/537316/233803958-773a100a-e13b-4a57-88a3-1a2ee250e0d3.png">

Reply:
Perfect, that sounds good.